server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -260,12 +266,14 @@ public void testExtensionsDiscovery() throws Exception {
                     "fakeClass2",
                     new ArrayList<String>(),
                     true
-                )
+                ),
+                List.of(new ExtensionDependency("uniqueId0", Version.fromString("2.0.0")))
             )
         );
         assertEquals(expectedExtensionsList.size(), extensionsOrchestrator.extensionIdMap.values().size());
         assertTrue(expectedExtensionsList.containsAll(extensionsOrchestrator.extensionIdMap.values()));
         assertTrue(extensionsOrchestrator.extensionIdMap.values().containsAll(expectedExtensionsList));
+        assertEquals(expectedExtensionsList.get(expectedExtensionsList.size() - 1), dependency.get(0));
```

Comment:
This works, but since we know we created a single-dependency list, can we just `get(0)` here?

Reply:
You are right. we do not need the `get(0)`, because we already create dependency as a single list and it is all we need.

Reply:
> You are right. we do not need the `get(0)`, because we already create dependency as a single list and it is all we need.

Correct. We could even ignore the list and actually create a `expectedDependency` object.